### ISSUE TITLE
Documentation out of date for customTypes

### ISSUE THREAD

**Issue by safaci2000** [jOOQ/jOOQ#7373]
### Expected behavior and actual behavior:

When running the code generation on jooq 3.10 I get the following warnings:

WARNING: DEPRECATION              : The <customTypes/> configuration element has been deprecated in jOOQ 3.10. Use <forcedTypes/> only, instead.

When Looking up the documentation for jooq 3.10:

https://www.jooq.org/doc/3.10/manual/code-generation/custom-data-types/

The example still directs the user to use customTypes.  

### Versions:

- jOOQ: 3.10
- Java: Any
- Database (include vendor): Postgres

**Comment by lukaseder** [comment]
Thanks for pointing this out. Eek, the "undocument" part of #5750 was missing, I guess. Will fix right away

**Comment by relgames** [comment]
Is there also a warning which should be deleted? 
Getting the following warning after I removed `customType`:
```
WARNING: Bad configuration for <forcedType/> JsonNode. No matching <customType/> found, and no matching SQLDataType found: <name>JsonNode</name><includeExpression>.*</includeExpression><includeTypes>JSONB</includeTypes><nullability>ALL</nullability><objectType>ALL</objectType>
```
My gradle config:
```
                        forcedTypes {
                            forcedType {
                                name = "JsonNode"
                                includeExpression = ".*"
                                includeTypes = "JSONB"
                                userType = "com.fasterxml.jackson.databind.JsonNode"
                                converter = "com.company.orders.db.PostgresJacksonBinding"
                            }
                        }
```

**Comment by lukaseder** [comment]
@relgames Yes, we could remove the reference from the warning, at least for those cases where users don't actually have any configured `<customType/>` references anymore.

If users do not migrate off the deprecated feature just yet, then message is still helpful, I think.

**Comment by lukaseder** [comment]
Fixing this through https://github.com/jOOQ/jOOQ/issues/10944
